recipes-domu/domu-image-android/files/meta-xt-images-extra/recipes-extended/android/android.bb

```diff
@@ -38,11 +33,12 @@ do_configure() {
 
 do_compile() {
     cd ${S}
+    SYS_ROOT="${S}/../recipe-sysroot-native/usr/bin"
 
     # run Android build in sane environment
     env -i HOME="$HOME" LC_CTYPE="${LC_ALL:-${LC_CTYPE:-$LANG}}" USER="$USER" \
-           PATH="${JAVA_HOME}/bin:$PATH" \
-           JAVA_HOME="${JAVA_HOME}" OUT_DIR_COMMON_BASE="${ANDROID_OUT_DIR_COMMON_BASE}" \
+           PATH="${SYS_ROOT}:${PATH}" \
```

Comment:
Please check that the sysroot you want is actually the native one, e.g. you add it to the PATH so native tools are visible, not those for target

Reply:
[Please see](https://github.com/openembedded/openembedded-core/blob/master/meta/conf/bitbake.conf#L378)